import-service/serverless.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-template-curly-in-string */
+import type { Serverless } from 'serverless/aws';
+
+const serverlessConfiguration: Serverless = {
+  service: {
+    name: 'import-service',
+  },
+  frameworkVersion: '2',
+  custom: {
+    webpack: {
+      webpackConfig: './webpack.config.js',
+      includeModules: true,
+    },
+  },
+  // Add the serverless-webpack plugin
+  plugins: ['serverless-webpack'],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs12.x',
+    profile: 'nodejs-in-aws-2',
+    region: 'eu-west-1',
+    stage: 'dev',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+    },
+    iamRoleStatements: [
+      {
+        Effect: 'Allow',
+        Action: 'S3:ListBucket',
+        Resource: 'arn:aws:s3:::arsiompeshkonodejsinaws2-product-import',
+      },
+      {
+        Effect: 'Allow',
+        Action: 's3:*',
+        Resource: 'arn:aws:s3:::arsiompeshkonodejsinaws2-product-import/*',
+      },
+    ],
+  },
+  functions: {
+    importProductsFile: {
+      handler: 'handlers/importProductsFile.handler',
+      events: [
+        {
+          http: {
+            method: 'get',
+            path: '/import',
+            cors: true,
```

Comment:
It is a good practice to describe the query params here as well, just to see you will use it within the code.